src/IconLists.js

```diff
@@ -101,6 +101,11 @@ const baseIcons = [
   'Search',
   'Settings',
   'SignLanguageContent',
+  'Sort',
+  'SortUp',
+  'SortDown',
+  'SortUpAlph',
+  'SortDownAlph',
```

Comment:
Should the naming use ascending/descending vs up/down, since it's more common when talking about sorting order?

Reply:
Yeah, makes sense. Thought about that aspect as well. Maybe it's more intuitive as it is in terms of DX.